src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingService.java

```diff
@@ -574,4 +568,26 @@ public DocumentInfo processHearingDocument(ListingData listingData, String caseT
             throw new DocumentManagementException(MESSAGE + caseTypeId, ex);
         }
     }
+
+    public boolean checkInvalidCharsForAllParties(ListingDetails listingDetails,
+                                                  String authToken, List<String> errors) {
+        try {
+            List<SubmitEvent> submitEvents = getListingHearingsSearch(listingDetails, authToken);
+            if (submitEvents != null) {
+                log.info(CASES_SEARCHED + submitEvents.size());
+                List<String> invalidCharErrors = InvalidCharacterCheck.areCharsForClaimantsRespValid(submitEvents);
+                if (CollectionUtils.isEmpty(invalidCharErrors)) {
+                    return true;
+                } else {
+                    errors.addAll(invalidCharErrors);
+                    return false;
+                }
+            }
+            return true;
+        }
+
```

Comment:
Are you introducing a new layout for try-catch blocks?
```
try {
...
} catch (Exception e) {
...
}
```

Also when you throw an exception do it like this
`throw new CaseCreationException(MESSAGE + listingDetails.getCaseId(), ex);`

Reply:
catch moved to previous line. used another class ReportException which has super class RunTimeException and takes ex as parameter.